digid_eherkenning/oidc/models/digid.py

```diff
@@ -36,48 +34,40 @@ class Meta:
 
     @property
     def oidcdb_username_claim(self) -> ClaimPath:
-        return [self.identifier_claim_name]
+        return self.bsn_claim
 
 
 class DigiDMachtigenConfig(OpenIDConnectBaseConfig):
-    # TODO: support periods in claim keys
-    vertegenwoordigde_claim_name = models.CharField(
-        verbose_name=_("vertegenwoordigde claim name"),
-        default="aanvrager.bsn",
-        max_length=50,
-        help_text=_(
-            "Name of the claim in which the BSN of the person being represented is stored"
-        ),
+    # TODO: these default claim names don't appear to be part of any standard...
+    representee_bsn_claim = ClaimField(
+        verbose_name=_("representee bsn claim"),
+        default=ClaimFieldDefault("urn:nl-eid-gdi:1.0:LegalSubjectID"),
+        help_text=_("Name of the claim holding the BSN of the represented user."),
     )
-    gemachtigde_claim_name = models.CharField(
-        verbose_name=_("gemachtigde claim name"),
-        default="gemachtigde.bsn",
-        max_length=50,
+    authorizee_bsn_claim = ClaimField(
+        verbose_name=_("representee bsn claim"),
+        default=ClaimFieldDefault("urn:nl-eid-gdi:1.0:ActingSubjectID"),
+        help_text=_("Name of the claim holding the BSN of the authorized user."),
+    )
+    mandate_service_id_claim = ClaimField(
```

Comment:
I'm not sure what the name of this field here should be, but the `help_text` mentions UUID while the field name itself uses `id`, is this intentional? In `eherkenning.py` this are two fields `mandate_service_id_claim` and `mandate_service_uuid_claim`.

Reply:
so for DigiD, it is an ID (semantically) which, as far as I can tell, happens to be a UUID4, but I haven't found conclusive documentation yet that it is guaranteed to be a UUID. DigiD machtigen knows only this single ID and it's super vague where it comes from.

in eherkenning, the ID is a `urn:<lots of stuff>`, and the UUID is indeed a UUID4. Both can be used to retrieve the service (or service instance).